tests/handlers/koji/test_rebuild_flatpak_application_on_module_ready.py

```diff
@@ -178,5 +180,174 @@ def test_event_state_updated_when_no_images_with_higher_rpm_nvr(self):
         self.assertEqual(db_event.state, EventState.SKIPPED.value)
         self.assertEqual(
             db_event.state_reason,
-            "There is no image can be rebuilt. message_id: 123",
+            "No images are impacted by the advisory.  message_id: 123",
         )
+
+    def test_get_module_name_stream_set(self):
+        build = {
+            "build_id": 1850907,
+            "epoch": None,
+            "extra": {
+                "typeinfo": {
+                    "module": {
+                        "modulemd_str": '---\ndocument: modulemd\nversion: 2\ndata:\n  name: ghc\n  stream: "9.2"\n  '
+                        "version: 3620211101111632\n  context: d099bf28\n  summary: Haskell GHC 9.2\n  "
+                        "description: >-\n    This module provides the Glasgow Haskell Compiler version 9.2.1\n",
+                        "name": "ghc",
+                        "stream": "9.2",
+                        "module_build_service_id": 13274,
+                        "version": "3620211101111632",
+                        "context": "d099bf28",
+                        "content_koji_tag": "module-ghc-9.2-3620211101111632-d099bf28",
+                    }
+                }
+            },
+            "id": 1850907,
+            "name": "ghc",
+            "nvr": "ghc-9.2-3620211101111632.d099bf28",
+        }
+        get_build = self._patch("freshmaker.kojiservice.KojiService.get_build")
+        get_build.return_value = build
+        get_cve_affected_build_nvrs = self._patch(
+            "freshmaker.errata.Errata.get_cve_affected_build_nvrs"
+        )
+        get_cve_affected_build_nvrs.return_value = ["ghc-9.2-3620211101111632.d099bf28"]
+        ret = self.handler._get_module_name_stream_set(self.event)
+        self.assertEqual(ret, {"ghc:9.2"})
+
+    @patch("freshmaker.odcsclient.create_odcs_client")
+    def test_prepare_data_for_compose(self, create_odcs_client):
```

Comment:
It seems like there's a lot of repeated code in this test and it becomes hard to read it. Any chance we can shorten this a bit? Consider using `parametrize` or maybe a for loop.

Reply:
The data has a small difference. Let me separate the test cases for readability.